.github/scripts/generate_binary_build_matrix.py

```diff
@@ -385,6 +387,7 @@ def generate_wheels_matrix(
                         "desired_cuda": translate_desired_cuda(
                             gpu_arch_type, gpu_arch_version
                         ),
+                        "use_split_build": "True",
```

Comment:
Or if you prefer to add it only if true, you can use something like `{**{original_dict_defines}, **({"use_split_build": "True"} if use_split_build else {})}`
```suggestion
                        "use_split_build": "True" if use_split_build else "False",
```

Reply:
I think having it there explicitly is reasonable especially if we make it the default.